src/components/text-editor/link-menu/limel-text-editor-link-menu.tsx

```diff
@@ -0,0 +1,150 @@
+import { Component, Prop, h, Event, EventEmitter, State } from '@stencil/core';
+import { EditorTextLink } from '../prosemirror-adapter/menu/types';
+import { Languages } from 'src/components/date-picker/date.types';
+import translate from '../../../global/translations';
+
+/**
+ * This component is a menu for editing a link in the text editor.
+ * It allows the user to input a title of a link and a link itself.
+ */
+
+@Component({
+    tag: 'limel-text-editor-link-menu',
+    shadow: true,
+    styleUrl: 'limel-text-editor-link-menu.scss',
+})
+export class TextEditorLinkMenu {
+    /**
+     * The link
+     */
+    @Prop()
+    public link: EditorTextLink;
+
+    /**
+     * Defines the language for translations.
+     * Will translate the translatable strings on the components.
+     */
+    @Prop({ reflect: true })
+    public language: Languages = 'en';
+
+    /**
+     * Open state of the dialog
+     */
+    @Prop()
+    public isOpen: boolean = false;
+
+    @State()
+    private invalidLink: boolean = false;
+
+    /**
+     * Emitted when the menu is closed from inside the component.
+     * (*Not* emitted when the consumer sets the `open`-property to `false`.)
+     */
+    @Event()
+    private cancel: EventEmitter<void>;
+
+    /**
+     * Emitted when the menu is saved from inside the component.
+     */
+    @Event()
+    private save: EventEmitter<void>;
+
+    /**
+     * Emitted when the user inputs new values for the link
+     */
+    @Event()
+    private linkChange: EventEmitter<EditorTextLink>;
+
+    public render() {
+        return [
+            <limel-input-field
+                label="Text"
+                value={this.link?.text || ''}
+                leadingIcon="open_book"
```

Comment:
```suggestion
                leadingIcon="text_cursor"
```

Reply:
⚡ 

(@john-traas You can use ⚡ to indicate that you've addressed the comment, without having to resolve the thread. In this case though, I'm pretty sure Kia doesn't mind me resolving the thread even if he hasn't had time to double check that you made the change that he meant, because there really isn't much room for misunderstanding in this case 😊)

Reply:
Ah, good to know. Thanks 😊

Reply:
😊 Personally I don't mind if you just mark as resolved neither. And if you use the <kbd>Commit suggestion</kbd> button directly in GitHub, it will mark the thread as "resolved" automatically. But when <kbd>Commit suggestion</kbd>  becomes disabled, it also means you've adressed the thing. So do whatever works for you. But commonly we use ⚡, specially if we address the threads it very quickly 😄